explorer/src/lib/components/block-details/BlockDetails.svelte

```diff
@@ -22,16 +22,17 @@
   /** @type {number} */
   let screenWidth = window.innerWidth;
 
+  /** @type {HTMLElement}*/
+  let blockList;
+
   onMount(() => {
     const resizeObserver = new ResizeObserver((entries) => {
       const entry = entries[0];
 
       screenWidth = entry.contentRect.width;
     });
 
-    resizeObserver.observe(
-      document.body.getElementsByClassName("details-list__definition")[0]
-    );
+    resizeObserver.observe(blockList.children[1]);
```

Comment:
I'm thinking this should be enough:

`resizeObserver.observe(blockList);`

<img width="1373" alt="Screenshot 2024-04-18 at 16 27 05" src="https://github.com/dusk-network/rusk/assets/34094428/c2d3f7e8-8d18-4142-b615-192a5de76bba">

<img width="1366" alt="Screenshot 2024-04-18 at 16 26 56" src="https://github.com/dusk-network/rusk/assets/34094428/dba5f97d-4cff-4211-ba42-bd02fd4e148d">

This will give the full hash on desktop, otherwise we are showing just a tiny bit.